build-tools/installers/create-installers.targets

```diff
@@ -119,6 +119,7 @@
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Mono.Options.dll" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Mono.Options.pdb" />
     <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)MULTIDEX_JAR_LICENSE" />
+    <_MSBuildFiles Include="$(MicrosoftAndroidSdkOutDir)Newtonsoft.Json.dll" />
```

Comment:
Will some SignCheck step fail in VS, if we are missing `Newtonsoft.Json.pdb`? Do we have another pipeline to fetch symbols now?